web/profiles/custom/os2loop/modules/os2loop_section_page/os2loop_section_page.module

```diff
@@ -22,3 +23,20 @@ function os2loop_section_page_help($route_name, RouteMatchInterface $route_match
     default:
   }
 }
+
+/**
+ * Implements hook_form_alter().
+ */
+function os2loop_section_page_form_alter(&$form, FormStateInterface $form_state, $form_id) {
```

Comment:
For consistency with other OS2Loop modules, this should be moved to a helper class with a `alterForm` method.

A comment telling what is actually done would be helpful. I suppose we're hiding some view display settings.